src/components/TrackInfo.vue

```diff
@@ -0,0 +1,301 @@
+<template>
+  <div class="track-info">
+    <img
+      v-if="image"
+      class="track-info-image"
+      :src="image"
+      alt=""
+    >
+    <div class="track-info-details">
+      <div
+        v-if="livestream"
+        class="track-info-livestream"
+      >
+        <div class="track-info-livestream-indicator">
```

Comment:
I made a component for the live indicator because it's in the main player as well